src/Neo/Ledger/MemoryPool.cs

```diff
@@ -275,7 +279,8 @@ private PoolItem GetLowestFeeTransaction(out Dictionary<UInt256, PoolItem> unsor
             }
         }
 
-        // Note: this must only be called from a single thread (the Blockchain actor)
+        // Note: this must only be called from a single thread (the Blockchain actor) and
+        // doesn't take into account conflicting transactions.
```

Comment:
If it doesn't take into account conflicting, why do we check on line 301? `  if (!CheckConflicts(tx, out List<PoolItem> conflictsToBeRemoved)) return VerifyResult.HasConflicts;`

Reply:
Because line 301 is a separate non-related method (`internal VerifyResult TryAdd(Transaction tx, DataCache snapshot)`). And this one is `internal bool CanTransactionFitInPool(Transaction tx)` and it checks whether there's a free space in the pool and (if not) whethere there's less prioritized transaction (with less fee). `CanTransactionFitInPool` must be light-weighted and it doesn't contain all the checks that `internal VerifyResult TryAdd(Transaction tx, DataCache snapshot)` has, it's OK.